pkg/apis/networking/v1/types.go

```diff
@@ -27,8 +27,8 @@ type NetworkMode string
 
 const (
 	NetworkModeBGP   = NetworkMode("BGP")
-	NetworkModeVlan  = NetworkMode("Vlan")
-	NetworkModeVxlan = NetworkMode("Vxlan")
+	NetworkModeVlan  = NetworkMode("VLAN")
```

Comment:
should rename variable to `NetworkModeVLAN`?

Reply:
`netlink.Vlan{}` is also be used in `vishvananda/netlink` package. I mean, it's a little bit more beautiful for a name of variable.